.github/workflows/CD.yml

```diff
@@ -639,9 +647,46 @@ jobs:
         env:
           AWS_SDK_LOAD_CONFIG: true
 
+  clean-ssm-package:
+    runs-on: ubuntu-latest
+    if: ${{ always() }}
+    needs: [ s3-release-validation-1, s3-release-validation-2, s3-release-validation-3, release-validation-ecs, release-validation-eks, release-checking ]
+    steps:
+      #Delete ssm package and binaries/artifacts which is created on s3 bucket for validation test regardless of these two scenarios:
+      #-if the validation test failed, then we will delete the SSM package since we don't want to roll out the unstable version
+      #-if the validation successes, then we will trigger the release SSM package workflow to replace all of this with the appropriate region
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.RELEASE_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.RELEASE_SECRET }}
+          aws-region: us-west-2
+
+      - name: Restore cached packages
+        id: cache_packages
+        uses: actions/cache@v2
+        with:
+          key: ${{ env.PACKAGE_CACHE_KEY }}
+          path: ${{ env.PACKAGING_ROOT }}
+
+      - name: Update SSM default version
+        if: steps.cache_packages.outputs.cache-hit == 'true'
+        run: ssm_package_name=${{ env.SSM_RELEASE_PACKAGE_NAME }} version=${{ needs.release-checking.outputs.version }}  tools/ssm/ssm_update_default_version.sh
```

Comment:
The script should probably mention that it's rolling back the default version instead of just updating it. From the name, it sounds like you're updating the default version to the version passed in.

Reply:
Reasonable. Fixed it in newest commit.